model/instance/lifecycle/destroy.go

```diff
@@ -39,7 +38,9 @@ func DestroyWithoutHooks(domain string) error {
 
 	// Deleting accounts manually to invoke the "account deletion hook" which may
 	// launch a worker in order to clean the account.
-	deleteAccounts(inst)
+	if err := deleteAccounts(inst); err != nil {
+		return err
```

Comment:
What happens then ? Do we have a pager notification ;) ?

Reply:
Not yet, but it will come in a next PR